update sizing cluster info (#8040)
fishtown-analytics/docs.getdbt.com: website/docs/guides/set-up-your-databricks-dbt-project.md

## Setting up Databricks Compute

When you run a dbt project, it generates SQL, which can run on All Purpose Clusters or SQL warehouses. We strongly recommend running dbt-generated SQL on a Databricks SQL warehouse. Since SQL warehouses are optimized for executing SQL queries, you can save on the cost with lower uptime needed for the cluster to run the queries. If you need to debug, you will also have access to a Query Profile. 

> We recommend using a serverless cluster if you want to minimize the time spent on spinning up a cluster and removing the need to change cluster sizes depending on workflows. If you use a Databricks serverless SQL warehouse, you still need to choose a [warehouse “size”](https://docs.databricks.com/aws/en/compute/sql-warehouse/create#configure-sql-warehouse-settings) (e.g. 2X-Small, X-Small, Small, Medium, Large, etc.). For more information on serverless SQL warehouses, see the [Databricks docs](https://docs.databricks.com/aws/en/compute/sql-warehouse/warehouse-behavior#sizing-a-serverless-sql-warehouse).

Reviewer: The Databricks doc uses the term "cluster size", should we also use the same term here for consistency? Also edited the [abbreviations](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#latin-abbreviations) used
```suggestion
We recommend using a serverless cluster if you want to minimize the time spent on spinning up a cluster and removing the need to change cluster sizes depending on workflows. If you use a Databricks serverless SQL warehouse, you still need to choose a [cluster size](https://docs.databricks.com/aws/en/compute/sql-warehouse/create#configure-sql-warehouse-settings) (for example, 2X-Small, X-Small, Small, Medium, Large). For more information on serverless SQL warehouses, see the [Databricks docs](https://docs.databricks.com/aws/en/compute/sql-warehouse/warehouse-behavior#sizing-a-serverless-sql-warehouse).
```

> We recommend using a serverless cluster if you want to minimize the time spent on spinning up a cluster and removing the need to change cluster sizes depending on workflows. If you use a Databricks serverless SQL warehouse, you still need to choose a [warehouse “size”](https://docs.databricks.com/aws/en/compute/sql-warehouse/create#configure-sql-warehouse-settings) (e.g. 2X-Small, X-Small, Small, Medium, Large, etc.). For more information on serverless SQL warehouses, see the [Databricks docs](https://docs.databricks.com/aws/en/compute/sql-warehouse/warehouse-behavior#sizing-a-serverless-sql-warehouse).

Reviewer: The Databricks doc uses the term "cluster size", I suggest we also use the same term for consistency. Also edited the [abbreviations](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#latin-abbreviations) used
```suggestion
We recommend using a serverless cluster if you want to minimize the time spent on spinning up a cluster and remove the need to change cluster sizes depending on workflows. If you use a Databricks serverless SQL warehouse, you still need to choose a [cluster size](https://docs.databricks.com/aws/en/compute/sql-warehouse/create#configure-sql-warehouse-settings) (for example, 2X-Small, X-Small, Small, Medium, Large). For more information on serverless SQL warehouses, see the [Databricks docs](https://docs.databricks.com/aws/en/compute/sql-warehouse/warehouse-behavior#sizing-a-serverless-sql-warehouse).
```

> We recommend using a serverless cluster if you want to minimize the time spent on spinning up a cluster and removing the need to change cluster sizes depending on workflows. If you use a Databricks serverless SQL warehouses, you still need to choose a [warehouse “size”](https://docs.databricks.com/aws/en/compute/sql-warehouse/create#configure-sql-warehouse-settings) (e.g. 2X-Small, X-Small, Small, Medium, Large, etc.). For more information on serverless SQL warehouses, see the [Databricks docs](https://docs.databricks.com/aws/en/compute/sql-warehouse/warehouse-behavior#sizing-a-serverless-sql-warehouse).

Author: ```suggestion
We recommend using a serverless cluster if you want to minimize the time spent on spinning up a cluster and removing the need to change cluster sizes depending on workflows. If you use a Databricks serverless SQL warehouse, you still need to choose a [warehouse “size”](https://docs.databricks.com/aws/en/compute/sql-warehouse/create#configure-sql-warehouse-settings) (e.g. 2X-Small, X-Small, Small, Medium, Large, etc.). For more information on serverless SQL warehouses, see the [Databricks docs](https://docs.databricks.com/aws/en/compute/sql-warehouse/warehouse-behavior#sizing-a-serverless-sql-warehouse).
```

Let’s [create a Databricks SQL warehouse](https://docs.databricks.com/sql/admin/sql-endpoints.html#create-a-sql-warehouse):

1. Click **SQL Warehouses** in the sidebar.

> 2. Click *Create SQL Warehouse*.

Reviewer: ```suggestion
2. Click **Create SQL Warehouse**.
```

3. Enter a name for the warehouse.
4. If using a serverless SQL warehouse, select a [cluster size](https://docs.databricks.com/aws/en/compute/sql-warehouse/warehouse-behavior#sizing-a-serverless-sql-warehouse) (2X-Small through 4X-Large) or leave the default, but ensure it suits your workload.
5. Accept the default warehouse settings or edit them.

> 6. Click *Create*

Reviewer: ```suggestion
6. Click **Create**.
```
